.github/workflows/deploy.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Installing dependencies
         run: yarn install --frozen-lockfile
       - name: Build lib
-        run: yarn run build
+        run: yarn compile && yarn compile:css
```

Comment:
what is the difference between the change and the new command?

Reply:
`yarn run build` never worked in this repo. Jeremy probably just copied it from a different repo. `yarn compile` actually compiles the code, and I modified `yarn compile:css` to compile the css once.